integr8ly/integreatly-operator
# add preflight checks

Add a few preflight checks:
- That required secrets for s3 exist
- That none of our to-be-installed products are currently installed

## Verification
- create a test namespace (without s3 secrets) and run operator
- Create installation CR
- See preflight fail regarding missing secret.
- Create secrets
- Install succeeds
- Create a second test namespace (with s3 secrets) and run operator
- See preflight fails regarding existing installed products
- Delete first CR and wait for deprovision of first install
- See preflight pass on second and install succeed.
